src/librustc_trans/back/link.rs

```diff
@@ -938,8 +938,10 @@ fn add_upstream_rust_crates(cmd: &mut Linker,
             Linkage::NotLinked |
             Linkage::IncludedFromDylib => {}
             Linkage::Static => {
+                let is_a_no_builtins_crate =
+                    attr::contains_name(&sess.cstore.crate_attrs(cnum), "no_builtins");
```

Comment:
Ah yeah something more readable is fine, I'd just want to deduplicate the string literal `"no_builtins"`